tests/performance/specs/admin-l10n.test.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * WordPress dependencies
+ */
+import { test } from '@wordpress/e2e-test-utils-playwright';
+
+/**
+ * Internal dependencies
+ */
+import { camelCaseDashes } from '../utils';
+
+const results = {
+	timeToFirstByte: [],
+};
+
+test.describe( 'Admin (L10N)', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
```

Comment:
To what extent does activating a specific theme matter for these two admin tests vs. using the default theme? Are you mainly just trying to maintain consistency from version to version?

Reply:
Yes, mainly for consistency